File: Python/FilmBioskop.py
class FilmBioskop:
    def __init__(self, id="", judul="", genre="", hargaTiket=0, gambar=""):
        self.__id = id
        self.__judul = judul
        self.__genre = genre
        self.__hargaTiket = hargaTiket
        self.__gambar = gambar

    # Getter & Setter
    def get_id(self): return self.__id
    def set_id(self, id): self.__id = id

    def get_judul(self): return self.__judul
    def set_judul(self, judul): self.__judul = judul

    def get_genre(self): return self.__genre
    def set_genre(self, genre): self.__genre = genre

    def get_hargaTiket(self): return self.__hargaTiket
    def set_hargaTiket(self, harga): self.__hargaTiket = harga

    def get_gambar(self): return self.__gambar
    def set_gambar(self, gambar): self.__gambar = gambar

File: Java/FilmBioskop.java
public class FilmBioskop {
    private String id;
    private String judul;
    private String genre;
    private int hargaTiket;
    private String gambar;

    public FilmBioskop() {}

    public FilmBioskop(String id, String judul, String genre, int hargaTiket, String gambar) {
        this.id = id;
        this.judul = judul;
        this.genre = genre;
        this.hargaTiket = hargaTiket;
        this.gambar = gambar;
    }

    public String getId() { return id; }
    public void setId(String id) { this.id = id; }

    public String getJudul() { return judul; }
    public void setJudul(String judul) { this.judul = judul; }

    public String getGenre() { return genre; }
    public void setGenre(String genre) { this.genre = genre; }

    public int getHargaTiket() { return hargaTiket; }
    public void setHargaTiket(int hargaTiket) { this.hargaTiket = hargaTiket; }

    public String getGambar() { return gambar; }
    public void setGambar(String gambar) { this.gambar = gambar; }
}